ReText/window.py

```diff
@@ -428,6 +434,26 @@ def dropEvent(e):
 		self.tabWidget.dropEvent = dropEvent
 		self.tabWidget.setTabBarAutoHide(globalSettings.tabBarAutoHide)
 
+	def initDirectoryTree(self, visible, path):
+		if visible:
+			self.fileSystemModel = QFileSystemModel(self.treeView)
+			self.fileSystemModel.setRootPath(path)
+			self.treeView.setModel(self.fileSystemModel)
+			self.treeView.setRootIndex(self.fileSystemModel.index(path))
+			self.treeView.setColumnHidden(1, True)
+			self.treeView.setColumnHidden(2, True)
+			self.treeView.setColumnHidden(3, True)
+			self.treeView.setHeaderHidden(True)
+			self.splitter.setSizes([self.width() / 5, self.width() * 4 / 5])
+		else:
+			self.splitter.setSizes([0, self.width()])
```

Comment:
This way the splitter is still visible, and it is possible to drag it to the right — which results in a blank space.

I think it's better to call `set.treeView.setVisible(False)` in this case — like we do for the tab's QSplitter when live preview is disabled.